signature/jws/conformance_test.go

```diff
@@ -0,0 +1,120 @@
+package jws
+
+import (
+	"crypto/elliptic"
+	"crypto/x509"
+	"reflect"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-core-go/testhelper"
+)
+
+var (
+	// signedAttributes for signing request
+	signedAttributes = signature.SignedAttributes{
+		SigningScheme: signature.SigningSchemeX509,
```

Comment:
Let's hard code the string here for testing purpose instead of calling the variable in the package.

Reply:
Done.